Requirement already satisfied: python-dateutil in c:\users\asus\desktop\innomatics_internship\debuging\myenv\lib\site-packages (1.5)
